src/logic/domains/workshopDetails.domain.ts

```diff
@@ -0,0 +1,26 @@
+export interface Workshop {
+    _id?: string;
+    no: ICheckBox; // Should be a date object
+    skillSet: string; // Should be a date object
+    shortTitle: string;
+
+    // advisor: string;
+    // type: string;
+    // currentBooking: WorkshopDetails;
+    // waitingList: Array<WorkshopDetails>
+    // attendedNotAttended: string; (not sure what this is)
+}
+
+export interface WorkshopDetails {
+    [studentId: string]: any;
```

Comment:
this looks like it should be workshopId, because it's an index I dont believe it means you can use any string when you have a WorkshopDetails object but anyone looking at it might get confused.